src/main/java/org/apache/accumulo/testing/randomwalk/multitable/BulkImport.java

```diff
@@ -98,18 +97,8 @@ public void visit(final State state, final RandWalkEnv env, Properties props) th
     }
     log.debug("Starting {} bulk import to {}", useLegacyBulk ? "legacy" : "new", tableName);
```

Comment:
Now that it is no longer used in the code, I think the "legacy bulk" options should be removed before this is merged